OneMore/Commands/File/Markdown/MarkdownConverter.cs

```diff
@@ -158,21 +195,43 @@ public MarkdownConverter RewriteTodo(IEnumerable<XElement> paragraphs)
 				{
 					var cdata = run.GetCData();
 					var wrapper = cdata.GetWrapper();
+					if (wrapper.FirstNode is XText)
+					{
+						cdata.Value = wrapper.GetInnerXml();
```

Comment:
This does absolutely nothing? Convert cdata to wrapper and then set cdata to wrapper XML? That's the same thing.